packages/reports/tests/integration/components/report-actions/multiple-format-export-test.js

```diff
@@ -109,18 +113,20 @@ module('Integration | Component | report actions - multiple-format-export', func
 
   test('filename', async function (assert) {
     assert.expect(1);
-
     const originalFlag = config.navi.FEATURES.exportFileTypes;
     config.navi.FEATURES.exportFileTypes = ['csv'];
 
     await render(TEMPLATE);
     await triggerEvent('.menu-trigger', 'mouseenter');
     await click($('.menu-content a:contains("CSV")')[0]);
-    assert.equal(
-      find('.export__download-link').getAttribute('download'),
-      'hyrule-news',
-      'The download attribute is set correctly'
-    );
+
+    assert
+      .dom('.export__download-link')
+      .hasAttribute(
+        'href',
+        /(?=.*?data.naviapp.io)(?=.*format=csv)(?=.*dateTime=)(?=.*csv&filename=Hyrule%20News-\d\d\d\d\d\d\d*T\d\d\d\d\d\d)/,
+        'csv export url is generated for the report'
+      );
```

Comment:
We test the url in the previous test, this test is for the `download` attr so maybe it should be changed to something like 
```suggestion
    assert.ok(
      /hyrule-news-\d\d\d\d\d\d\d*T\d\d\d\d\d\d)/.test(find('.export__download-link').getAttribute('download')),
      'The download attribute is set correctly'
    );
```